### PR TITLE
Add command to compare latest manifests

### PR DESCRIPTION
**Pull request by ggilder**
If two directories already have current manifest files, we can now
compare the latest manifest in each without rescanning the entire
directory contents.

@faun

### CODE REVIEW

File: bitrot.go [hunk 1]
```diff
@@ -219,6 +225,37 @@ func (cmd *Compare) Execute(args []string) (err error) {
...
 
+func (cmd *CompareLatestManifests) Execute(args []string) (err error) {
+	assertNoExtraArgs(&args, cmd.logger)
+	oldPath := pathString(cmd.Arguments.Old)
+	newPath := pathString(cmd.Arguments.New)
+
+	oldManifest := LatestManifestFileForPath(oldPath)
+	newManifest := LatestManifestFileForPath(newPath)
+
+	if oldManifest == nil {
```
Comment (faun) [inline bitrot.go]: Can you extract this duplication into a shared function?

**Comment by faun** [comment]
See my comments and :+1:

File: bitrot.go [hunk 2]
```diff
@@ -219,6 +225,37 @@ func (cmd *Compare) Execute(args []string) (err error) {
...
+		cmd.logger.Printf("No existing manifest for %s\n", newPath)
+		return nil
+	}
+
+	comparison := CompareManifests(oldManifest.Manifest, newManifest.Manifest)
+	cmd.logger.Printf(manifestComparisonReportString(comparison))
+
+	flagged := len(comparison.FlaggedPaths)
+	if flagged > 0 {
+		// TODO: want to use Fatalf here but can't seem to catch it in tests
```
Comment (ggilder) [inline bitrot.go]: LOL. see https://github.com/ggilder/bitrot/issues/22

File: bitrot.go (hunk 1 above)
Comment (ggilder) [inline bitrot.go]: I can, but the overhead/boilerplate is making me hesitate. I think the best compromise might be to make the formatting string a constant, to at least remove the string literal duplication.

Comment (ggilder) [inline bitrot.go]: But, I think I want to postpone that and fix all the string literals later on. Lots of duplication there already.

**Pull request merged**